src/exambank/exambank.py

```diff
@@ -0,0 +1,112 @@
+from io import BytesIO
+from pathlib import Path
+
+from pymupdf import Matrix
+from pymupdf import open as p_open
+from pypdf import PdfReader, PdfWriter
+from reportlab.lib import colors
+from reportlab.lib.pagesizes import A4
+from reportlab.lib.units import inch
+from reportlab.pdfgen import canvas
+
+# param to describe margin for exam generation text
+border = 20
```

Comment:
let's use SCREAM CASE FOR CONSTANTS
```suggestion
BORDER = 20
```